test/upload_test_helpers.js

```diff
@@ -134,8 +143,31 @@ module.exports = {
     }, config);
   },
 
+  createWebpackConfigAlt({ config, s3Config } = {}) {
+    return _.extend({
+      entry: ENTRY_PATH,
+      module: {
+        loaders: [{
+          test: /\.png/,
+          loader: `file-loader?name=${ASSET_FILE_NAME_PATTERN}?publicPath=${ASSET_PATH}&outputPath=${ASSET_OUTPUT_RELATIVE_PATH}`,
+        }, {
+          test: /\.css$/,
+          loader: ExtractTextPlugin.extract('css-loader'),
+        }],
+      },
+      plugins: [
+        new ExtractTextPlugin('styles/styles.css'),
+        generateS3Config(s3Config),
+      ],
+      output: {
+        path: OUTPUT_PATH_ALT,
+        filename: `${OUTPUT_FILE_NAME}-[hash]-${+new Date()}.js`,
+      },
+    }, config);
+  },
+
   runWebpackConfig({ config }) {
-    this.createOutputPath();
+    this.createOutputPath(config.output.path);
 
     return new Promise((resolve, reject) => {
```

Comment:
It would be good if you could add comprehensive description in README.md

Reply:
Updated readme